src/bot/commands/public/settings.command.ts

```diff
@@ -1,198 +1,43 @@
-  getSettingsMenuMessage,
-  goBackButton,
-  selectYourState,
-  settingsSet,
-  settingsSubmitMessage,
-  turnOffChatWhileAlarmButton,
-  warnRussianLanguageButton,
-} from '../../../message';
-import { alarmChatService } from '../../../services';
-import type { BooleanChatSettings, GrammyContext, GrammyMiddleware, State } from '../../../types';
-import { onlyAdmin } from '../../middleware';
-import { MiddlewareMenu } from '../../middleware-menu.menu';
-
-import { dynamicLocationMenu } from './air-raid-alarm/locations-menu-generator';
-
-const toggleSetting = async (context: GrammyContext, key: keyof BooleanChatSettings) => {
-  context.chatSession.chatSettings[key] = !context.chatSession.chatSettings[key];
-  const newText = getSettingsMenuMessage(context.chatSession.chatSettings);
-
-  if (context.msg?.text !== newText) {
-    await context.editMessageText(newText, { parse_mode: 'HTML' });
-  }
-
-  const featureEnableText = '✅ Це налаштування тепер увімкнено';
-  const featureDisableText = '⛔️ Це налаштування тепер вимкнено';
-
-  const newSettingAlertText = key.startsWith('enable')
-    ? context.chatSession.chatSettings[key]
-      ? featureEnableText
-      : featureDisableText
-    : context.chatSession.chatSettings[key]
-    ? featureDisableText
-    : featureEnableText;
-
-  await context.answerCallbackQuery({
-    text: newSettingAlertText,
-  });
-};
-
-const isStateSelected: GrammyMiddleware = (context, next) => {
-  const { state } = context.chatSession.chatSettings.airRaidAlertSettings;
-
-  if (!state) {
-    return context.answerCallbackQuery({
-      text: selectYourState,
-      show_alert: true,
-    });
-  }
-
-  return next();
-};
-
-const isAlarmNow: GrammyMiddleware = async (context, next) => {
-  const isAlarm = alarmChatService.isAlarmNow(context.chatSession.chatSettings.airRaidAlertSettings.state || '');
-  if (isAlarm) {
-    await context.answerCallbackQuery({
-      text: blockWhenAlarm,
-      show_alert: true,
-    });
-  } else {
-    return next();
-  }
-};
-
+import { environmentConfig } from '../../../config';
+import { redisService } from '../../../services';
+import type { GrammyMiddleware, Session } from '../../../types';
+// Todo change messages with needed text
 export class SettingsCommand {
-  /**
-   * @param {State[]} airRaidAlarmStates
-   * */
-  settingsMenuObj: MiddlewareMenu | null;
-
-  settingsDescriptionObj: MiddlewareMenu | null;
-
-  settingsAirRaidAlertObj: MiddlewareMenu | null;
-
-  constructor(private airRaidAlarmStates: State[]) {
-    this.settingsMenuObj = null;
-    this.settingsDescriptionObj = null;
-    this.settingsAirRaidAlertObj = null;
-  }
+  middleware(): GrammyMiddleware {
+    return async (context) => {
+      const isChatPrivate = context.chat?.type === 'private';
+      const userId = context.from?.id.toString() ?? '';
+      const chatId = context.chat?.id.toString() ?? '';
+      const linkToWebView = `Відкрити налаштування: ${environmentConfig.WEB_VIEW_URL}`;
```

Comment:
Move this message in `message.ts`.
Make it ukrainian.
Match it with our style (add an emoji, match text design).
If you need, you can use ChatGPT for it.